Release_Notes.md

```diff
@@ -14,11 +14,58 @@ See the License for the specific language governing permissions and<BR>
 limitations under the License.*_<BR>
 
 # **Release Notes** #
-_*Last Update: 2024/01/18*_
+_*Last Update: 2024/05/02*_
 
-## Rev 1p0 ##
+## Rev 1p1 ##
 
-### Rev 1p0 release date: 2024/01/18 ###
+#### Rev 1p1 release date: 2024/05/03 ####
```

Comment:
should this be `02` to match the other release dates provided?

Reply:
Can you clarify what `02` refers to? Is this a reference to the heading format or to the actual date value?
I updated heading formats to minimize future formatting issues when moving 'current' releases to 'previous releases' section.
Also, the actual final 1.1 release date remains TBD :(

Reply:
apologies; it refers to the date. On line 17 there is a date: 2024/05/02. I queried if this date on line 21 should match.

Reply:
ah, I see. I'll revise these dates again before we finalize the PR, as this will probably be the last PR before we tag 1.1.
I've been keeping the 'last update' date as the actual date that I modified the file, and the release date as our speculative date of tagging 1.1, which is why there was a discrepancy. I'll just keep these aligned in the future.

Reply:
Updated the dates for 7/2/2024